tox.ini

```diff
@@ -10,7 +10,8 @@ deps =
     mock
     pytest
     pytest-cov
-commands =
+    flake8
+commands = flake8
```

Comment:
Speaking of bikeshredding, could you add a newline before `flake8` here? ;) Almost read over it if it wasn't for the diff highlighting